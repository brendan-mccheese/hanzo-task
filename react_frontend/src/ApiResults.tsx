import * as React from "react";

interface ApiResultsProps {
    readonly helloMessage?: string;
    readonly postgresConn?: string;
    readonly redisConn?: any;
}
export const ApiResults = ({helloMessage, postgresConn, redisConn}: ApiResultsProps) => (
    <div>
        <h2 data-e2e-id="api-title">API Results</h2>
        <ul>
            <li>
                <span>Hello Message: </span>
                <span>{helloMessage ?? "No message received"}</span>
            </li>
        </ul>
    </div>
)