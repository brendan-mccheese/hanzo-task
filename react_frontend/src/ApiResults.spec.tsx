import React from "react";
import renderer from 'react-test-renderer';
import { ApiResults } from "./ApiResults";

describe("ApiResult", () => {
    test("renders", () => {
        const component = renderer.create(<ApiResults />)
        expect(component).toMatchSnapshot();
    });
});