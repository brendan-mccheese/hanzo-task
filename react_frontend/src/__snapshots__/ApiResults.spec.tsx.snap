// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ApiResult renders 1`] = `
<div>
  <h2>
    API Results
  </h2>
  <ul>
    <li>
      <span>
        Hello Message: 
      </span>
      <span>
        No message received
      </span>
    </li>
  </ul>
</div>
`;
